source/mongorestore.txt

```diff
@@ -623,6 +625,11 @@ Options
    ``<time_t>`` is the seconds since the UNIX epoch, and ``<ordinal>``
    represents a counter of operations in the oplog that occurred in the
    specified second.
+
+   .. warning::
+
+      ``oplogLimit`` might cause corruption and inconsistencies in the
```

Comment:
I suggest phrasing like, "use this option with caution, since manually specifying oplog entries to apply might cause corruption and inconsistencies in the restored data". We can use the same phrasing for this oplogLimit as well as oplogFile.